### ISSUE TITLE
Some nullable values are not properly handled

### ISSUE THREAD

**Issue by Keyacom** [beastmatser/aiopokeapi#70]
**Describe the bug**
A TypeError happens when there's some resource on the PokéAPI that contains a `null` value that is not yet handled by `aiopoke`. An example could be anything that is Fairy-type because its `move_damage_class` is `null`.

**To Reproduce**
1. Create a dummy Python script like this and run it:
    ```py
    import asyncio
    from aiopoke import AiopokeClient
    
    async def get_type(name: str):
        async with AiopokeClient() as client:
            return await client.get_type(name)
    
    print(asyncio.run(get_type('fairy')))
    ```

**Expected behavior**
A `NaturalGiftType` object for the Fairy type should be printed.

**Additional context**
Specifications on which values should be nullable are in the [schemas](https://github.com/PokeAPI/api-data/tree/master/data/schema/v2), for example, `type.move_damage_class`.

**Comment by Keyacom** [comment]
Seems like `type.move_damage_class` was already fixed in #62, but the fix does not yet exist in the PyPI release.

**Comment by beastmatser** [comment]
After some attempts I was able to generalise the problem (cad9614304a5f2a9aedab5bafc85e3d069ad347c). Every null value gets 'caught' and is then replaced with an empty dictionnary. If an object recieves no keyword arguments, then it will return `None`. So every case should be handled now. I sort of knew there were other cases, but didn't feel like looking too hard, since its kinda boring. Thanks for bringing this to my attention. I am going to close this issue, please reopen if you find a bug or a unintentional behaviour.
